ipld/hamt/Cargo.toml

```diff
@@ -29,3 +29,9 @@ murmur = ["murmur3"]
 
 [dev-dependencies]
 hex = "0.4.2"
+criterion = "0.3.1"
```

Comment:
Not that is matters but the latest version `0.3.3` I believe.

Reply:
patch versions get automatically bumped to the most up to date (if you look inside of the `Cargo.lock` you'll see the version is actually 0.3.3, but I will update this for peace of mind